docs/get-started/tools/shimmer-bridge.md

```diff
@@ -0,0 +1,350 @@
+
+:::
+
+### 5. Transfer the Tokens
+
+- Click "**Transfer**". Your wallet will ask you to approve the token spending and confirm the transaction
+- After your confirmation, a popup will occur; if you click on the link, you will be forwarded to [LayerZero Scan](https://layerzeroscan.com/) to observe the detailed progress of your bridge interaction.
+- The ShimmerBridge app will also display the progress of your transaction, including an estimated period till the bridging is completed in the "transactions" box that appears after confirmation.
+- Once the status changes to "completed", your tokens have arrived in your wallet.
+- Please make sure you have imported the correct token into your wallet on the destination chain to see it.
+
+## Troubleshooting
+
+Conditions that will hinder your transfer will be indicated in the "Action button" as follows:
+
+### Insufficient Balance
+
+- You entered a higher amount of tokens than you hold in your account.
+    -  Reduce the amount you want to transfer or add more tokens to your wallet.
+
+### Not Enough Native for Gas
+
+ - You must own more gas on the source chain to pay the required fees.
+    -  You need to send more of the source chain's native gas token to your wallet to resolve this.
+  
+### Select Different Chain
+
+- You have selected an incompatible selection of source and destination chain. It is only possible to bridge in and out of Shimmer EVM with ShimmerBridge; you cannot send from ETH to Avalanche, for example.
+    - Send either tokens into Shimmer EVM or send them from Shimmer EVM out to another chain.
+
+### Limit Exceeded
+
+- This can occur for bridgeable tokens that exist on several networks (ETH/WETH, USDT, and USDC).
+- These tokens can enter Shimmer EVM from different source chains but are represented as one token on Shimmer EVM. 
+If you aim to send these tokens out of Shimmer EVM, the LayerZero contracts on each chain can only payout the amount of these tokens that are currently locked in the contract, so you cannot send more assets back than currently are available to release. This would otherwise exceed the limits of that contract.
+ 
+#### Example
+
+- On Ethereum, 100 USDT are locked in the Bridge contract (from users that did send this USDT from ETH to Shimmer EVM)
+- On Avalanche, also 100 USDT are locked in the Bridge contract (from users that did send this USDT from Avalanche to Shimmer EVM)
+- You hold 150 USDT in your Wallet on Shimmer EVM
+- If you try to send this 150 USDT to Avalanche, it will exceed the amount currently locked in the Avalanche USDT Bridge contract; therefore, you can't execute such a transfer.
+- Reduce the amount you attempt to send until it is below the Limit.
+
+For any further questions or support, please visit the [LayerZero Discord](https://discord-layerzero.netlify.app/discord).
+
+## ShimmerBridge Token Addresses
+
+### LayerZero ShimmerBridge Wrapped Tokens on Shimmer Evm
```

Comment:
I think we should format the rest of the page different:

- [ ] Make it linkable: For every ecosystem use a level 4 header. That way we can easily people to the correct section
- [ ] I would probably not wrap it in info admonitions

Reply:
Linking the networks is a great addition, I added that